### PR TITLE
DDP-4709: Testboston styling - consent fix

### PR DESCRIPTION
**Pull request by fat23cat**

### CODE REVIEW

File: study-builder/studies/testboston/i18n/en.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   "consent": {
     "name": "Consent Form",
-    "title": "Consent Process<a class="button button_download" href="consent.pdf" download><img src="assets/images/download.svg" alt="Download file" class="activity-header__icon">Download this document</a>",
+    "title": """Consent Process<a class="button button_download" href="consent.pdf" download><img src="assets/images/download.svg" alt="Download file" class="activity-header__icon">Download this document</a>""",
```
Comment (fat23cat) [inline study-builder/studies/testboston/i18n/en.conf:9]: It looks like we should put the title content in the additional quotes due to inner quotes. Activity contains invalid HTML:
image

**Comment by yufengwng** [comment]
@fat23cat I have fixed this in one of my other PRs (somewhere in #258).

**Review by MocanaAtBroad — APPROVED** [review]

**Comment by fat23cat** [comment]
> @fat23cat I have fixed this in one of my other PRs (somewhere in #258).

So, should I close my PR?

**Comment by yufengwng** [comment]
👍 I think so :)

**Pull request closed without merging**